src/AstroBase.jl

```diff
@@ -485,4 +487,85 @@ function xy06(jd1, jd2)
     sec2rad((xpr + (xyls[1] + xypl[1]) / 1e6)), sec2rad(ypr + (xyls[2] + xypl[2]) / 1e6)
 end
 
+"""
+    nutation_00a(jd1, jd2)
+
+Returns luni-solar and planetary nutations for a given 2 part Julian date(TT).
+
+# Example
+
+```jldoctest
+julia> nutation_00a(2.4578265e6, 0.30440190993249416)
+(-3.7589903391357206e-5, -3.6659049617818334e-5)
+```
+"""
+function nutation_00a(jd1, jd2)
+    U2R = sec2rad(1/1e7)
+    t = ((jd1 - J2000) + jd2) / DAYS_PER_CENTURY
+
+    el = mean_anomaly(luna, t)
+    f  = mean_longitude_minus_lan(luna, t)
+    om = mean_longitude_ascending_node(luna, t)
+    apa  = general_precession_in_longitude(t)
+    alme = mean_longitude(mercury, t)
+    alve = mean_longitude(venus, t)
+    alea = mean_longitude(earth, t)
+    alma = mean_longitude(mars, t)
+    alju = mean_longitude(jupiter, t)
+    alsa = mean_longitude(saturn, t)
+    alur = mean_longitude(uranus, t)
+
+    elp = sec2rad(mod((@evalpoly t 1287104.79305 129596581.0481 -0.5532 0.000136 -0.00001149), ARCSECONDS_IN_CIRCLE))
+    d   = sec2rad(mod((@evalpoly t 1072260.70369 1602961601.2090 -6.3706 0.006593 -0.00003169), ARCSECONDS_IN_CIRCLE))
+
+    al   = mod(2.35555598 + 8328.6914269554 * t, 2pi)
```

Comment:
Use mod2pi, also below

Reply:
`mod2pi` was giving errors when tested with `ERFA.nut00a` :worried:

Reply:
Please, explain what kind of errors

Reply:
oh, I tried to generate the errors again, it's fixed, I guess it was due to another expression which I fixed together with removing `mod2pi` so I was in an impression that `mod2pi `'s regularized output might be of problem.
It's solved. :smile: